vxyowsup/whatsapp.py

```diff
@@ -0,0 +1,32 @@
+# -*- test-case-name: vumi.transports.whatsapp.tests.test_whatsapp -*-
+from twisted.internet import defer
+
+from vumi.transports.base import Transport
+from vumi.config import ConfigText
+from vumi import log
+
+
+class WhatsAppTransportConfig(Transport.CONFIG_CLASS):
+    extra_config = ConfigText('Some Config var', default='foo',
+                              static=True)
+
+
+class WhatsAppTransport(Transport):
+
+    CONFIG_CLASS = WhatsAppTransportConfig
+    transport_type = 'whatsapp'
+
+    def setup_transport(self):
+        config = self.get_static_config()
+        log.info('Transport starting with: %s' % (config.extra_config,))
+        return defer.succeed(1)
+
+    def teardown_transport(self):
+        return defer.succeed(1)
```

Comment:
I think there's no real need to return `defer.succeed(...)` in `setup_transport` and `teardown_transport`?